src/services/edusharing/services/EduSharingConnector.js

```diff
@@ -15,7 +15,7 @@ const ES_METADATASET =
 const ES_ENDPOINTS = {
 	AUTH: `${Configuration.get('ES_DOMAIN')}/edu-sharing/rest/authentication/v1/validateSession`,
 	NODE: `${Configuration.get('ES_DOMAIN')}/edu-sharing/rest/node/v1/nodes/-home-/`,
-	SEARCH: `${Configuration.get('ES_DOMAIN')}/edu-sharing/rest/search/v1/queriesV2/-home-/${ES_METADATASET}/ngsearch/`,
+	SEARCH: `${Configuration.get('ES_DOMAIN')}/edu-sharing/rest/search/v1/queries/-home-/${ES_METADATASET}/ngsearch/`,
```

Comment:
this change needs QA for all types of content

Reply:
The EduSharingConnector will be merged from main after PR https://github.com/hpi-schul-cloud/schulcloud-server/pull/3810 is done. So no review needed at this point in first place. Sorry for the trouble.